### ISSUE TITLE
add support for remote log server / client

### ISSUE THREAD

**Issue by bodsch** [bodsch/ansible-syslog-ng#4]
for server example
```
source net { udp(); };
destination remote { file("/var/log/remote/${FULLHOST}"); };
log { source(net); destination(remote); };
```

**Comment by bodsch** [comment]
for client example (from https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.16/administration-guide/12)
```
destination d_network { network("10.1.2.3" transport("udp")); };
``` 
```
destination d_network {
    syslog("10.1.2.3" transport("tcp"));
};
```
```
@version: 3.16
@include "scl.conf"
source s_local {
    system(); internal();
};
destination d_syslog_tcp {
    syslog("192.168.1.1" transport("tcp") port(2010));
};
log {
    source(s_local);destination(d_syslog_tcp);
};
```

**Comment by bodsch** [comment]
fixed with [1.6.0](https://github.com/bodsch/ansible-syslog-ng/releases/tag/1.6.0)
